packages/i18n/locales/en.json

```diff
@@ -319,6 +320,7 @@
       },
       "codeBox": {
         "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version. <link>Understand EOL support.</link>",
+        "ltsVersionFeaturesNotice": "Want new features sooner? Get <link>Latest Node.js</link> instead and try the latest improvements on Node!",
```

Comment:
```suggestion
        "ltsVersionFeaturesNotice": "Want new features sooner? Get the <link>Latest Node.js</link> instead and try the latest improvements on Node!",
```

Reply:
Not easy to do so with the current architecture of how these client-sided components work. We can address in a future change.